core/loadpoint.go

```diff
@@ -1450,6 +1473,42 @@ func (lp *LoadPoint) Update(sitePower float64, cheap bool, batteryBuffered bool)
 		err = lp.setLimit(targetCurrent, required)
 	}
 
+	// WakeUp checks
+	if (!lp.vehicleWakeUpCalled || !lp.chargerWakeUpCalled) && mode != api.ModeOff && lp.chargeCurrent > 0 && lp.vehicleSoc < 99 && lp.chargePower == 0 && lp.enabled && lp.activeDuration.Minutes() > 5 {
+		lp.log.DEBUG.Printf("sleeping? Mode:%s LPStatus:%s Active:%.2fm SOC:%f Current:%f Power:%f ", mode, lp.status, lp.activeDuration.Minutes(), lp.vehicleSoc, lp.chargeCurrent, lp.chargePower)
+		// check if wakup called
+		if !lp.vehicleWakeUpCalled {
+			// call the Vehicle WakeUp if available
+			if vs, ok := lp.vehicle.(api.CallWakeUp); ok {
+				if err := vs.WakeUp(); err == nil {
+					lp.log.DEBUG.Printf("vehicle WakeUp API called")
+					lp.vehicleWakeUpCalled = true
+				}
+			} else {
+				// to avoid that the API will be checked every run if the veicle don't support the WakeUp call
+				lp.vehicleWakeUpCalled = true
+			}
+		}
+		if !lp.chargerWakeUpCalled {
+			// call the Charger WakeUp if available
+			if c, ok := lp.charger.(api.CallWakeUp); ok {
+				if err := c.WakeUp(); err == nil {
+					lp.log.DEBUG.Printf("charger WakeUp called")
+					lp.chargerWakeUpCalled = true
+				}
+			} else {
+				lp.chargerWakeUpCalled = true
+			}
+
+		}
+	}
+	// reset WakeUpCalled then car is charging again
+	if lp.chargePower > 0 && (lp.chargerWakeUpCalled || lp.vehicleWakeUpCalled) {
```

Comment:
Besser an den Status C hängen?

Reply:
Leider ist der Status C auch wenn er nicht läd aber laden könnte

Reply:
Nein, das ist Status B2. Oder wie evcc sagen würde: Status B & enabled.
Status C wäre SEHR merkwürdig, da die Karre dann dauernd unter Spannung stehen würde weil das Schütz der WB angezogen ist.

Reply:
Ja stimmt - war nur in meiner democonmfig falsch.
Hab ich geändert